### ISSUE TITLE
Add birthday reminder

### ISSUE THREAD

**Issue by mikjori** [rv178/bday#6]
Would it be possible to implement some kind of birthday reminder whenever you boot up your PC? Like a notification on startup or something similar.

**Comment by rv178** [comment]
Yep that is possible. It does have a `bday td` command which shows today's birthdays. Maybe I could add a command runner to it aswell?

**Comment by mb6ockatf** [comment]
You can just add `bday td` to your `.bashrc`, or to your `prompt`.
If you want a some kind of graphical reminder, then it really depends on OS, isn't it?

**Comment by rv178** [comment]
> You can just add `bday td` to your `.bashrc`, or to your `prompt`. If you want a some kind of graphical reminder, then it really depends on OS, isn't it?

yeah

**Comment by rv178** [comment]
I was thinking of adding a command runner but I think it would be bloat? not sure.

**Comment by mb6ockatf** [comment]
> I was thinking of adding a command runner but I think it would be bloat? not sure.

Bloat fr

**Comment by mb6ockatf** [comment]
I guess this can be closed as will never be implemented

**Comment by rv178** [comment]
Yes. Surprised to see this thread still active lol
